tests/pages/firefox/all.py

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage
+
+
+class FirefoxAllPage(FirefoxBasePage):
+
+    _url = '{base_url}/{locale}/firefox/all'
+
+    _search_input_locator = (By.ID, 'language-search-q')
+    _submit_button_locator = (By.CSS_SELECTOR, '#language-search button')
+    _build_rows_locator = (By.CSS_SELECTOR, '#builds table > tbody > tr')
+
+    @property
+    def displayed_builds(self):
+        return len([el for el in self.find_elements(
+                   self._build_rows_locator) if el.is_displayed()])
+
+    @property
+    def english_builds(self):
+        return len([el for el in self.find_elements(self._build_rows_locator)
+                   if 'english' in el.get_attribute('data-search')])
+
+    @property
+    def only_english_builds_are_displayed(self):
+        return self.english_builds == self.displayed_builds
```

Comment:
I'm a bit wary that this is doing a lot of DOM querying. It might be a nicer idea to only get the `english_builds` length once, instead of needing a repeated query. Not sure about the best approach here to do this. Thoughts?